### DISCUSSION TITLE
How to name chunks in the channel?

### DISCUSSION THREAD

**Discussion by Lunana0110** [nextflow-io/nextflow#4393] [Q&A]
I have a txt including a list of s3 path, and each 4 fastq files are from one sample. I would like to treat 4 fastqs as a whole and do qc and trimming etc.. I tried to use splitText(4) but "fastqc" seems not be able to take care of them... is it better to cat them? Any suggestions?

**Comment by mribeirodantas** [comment]
Could you please share the format in which the data is organized in your txt file? A CSV or TSV file with two columns, one for sample id and another for S3 paths should be enough to do what you're looking for. Read the content into Nextflow, group by sample id, and so on (check Nextflow operators [here](https://www.nextflow.io/docs/latest/operator.html)).
